pkg/sloop/webserver/webfiles.go

```diff
@@ -41,14 +37,11 @@ func readWebfile(filepath string, fs *afero.Afero) ([]byte, error) {
 // calling ReadWebfile ().
 func getTemplate(templateName string, _ []byte) (*template.Template, error) {
 	fs := afero.Afero{Fs: afero.NewOsFs()}
-	data, err := readWebfile((path.Join(prefix, templateName)), &fs)
-	if err != nil {
-		return nil, err
-	}
-	newTemplate := template.New(templateName)
-	newTemplate, err = newTemplate.Parse(string(data))
-	if err != nil {
-		return nil, err
+	data, err := readWebfile(templateName, &fs)
+	if err == nil {
```

Comment:
the usual pattern of `if err != nil {` should be followed unless strong reason not too. It leaves the main line code at the least indentation